src/main/java/org/apache/maven/plugins/changes/announcement/AnnouncementMojo.java

```diff
@@ -477,6 +477,7 @@ public void execute() throws MojoExecutionException {
             }
 
             if (issueManagementSystems.contains(TRAC)) {
+                getLog().warn("TRACT integration is prepared for remove in next major version, lack of maintainers");
```

Comment:
TRAC, TRACT, or Track?

Reply:
should be `TRAC` 😄 fixed

Reply:
Tractor 🚜

Reply:
Please fix the PR title as well.

Reply:
summary - jira, pr and commit also fixed 😄

Reply:
It is Trac

Reply:
I like a code review even with simple task ...